axiom/nr_distributed_trace.h

```diff
@@ -12,6 +12,8 @@
 #include "util_time.h"
 #include "util_object.h"
 
+#define NR_TRACE_ID_SIZE 32
```

Comment:
I don't think this name is clear - what NR trace spec says it should be 32?  This is not a NR related specification.  Perhaps `W3C_TRACE_ID_SIZE`?

Reply:
> This is not a NR related specification.

Nor is it a W3C specification. The W3C specification is for headers, which we already set to 32 character is a different place in the code. The NR Collector will accept a trace-id up to 32 characters and will prepend 0s to anything less than 32 characters. As such, the NR traceid size is 32.

Reply:
The purpose of the PR is W3C compatible trace ids so it seems this is a W3C driven decision.

Reply:
Also the default length we create is 16 characters.  Is that not the "NR" trace id length by definition?

Reply:
The purpose is OTLP Collector compatible trace ids. And New Relic backend full trace id size is 32. 32 is the full size that our NR Collector expects.

Reply:
As I understand the nomenclature "NR" is the namespace for this define - so the remaining part is "TRACE_ID_SIZE". Perhaps at a minimum we could have a comment to explain what this size is compared to `NR_GUID_SIZE`([code](https://github.com/newrelic/newrelic-php-agent/blob/6500c76c7515468e61ef634c2a466344bce6cf53/axiom/nr_guid.h#L16)) which also talks about connecting traces.

Reply:
added a comment in [215aa0e](https://github.com/newrelic/newrelic-php-agent/pull/929/commits/215aa0e7ed2898a181b124004dd612da490a4e01)